### PR TITLE
fix(display-name): do not default name to placeholder name

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: modules/UI/shared_video/SharedVideoThumb.js [hunk 1]
```diff
@@ -65,9 +65,9 @@ SharedVideoThumb.prototype.createContainer = function(spanId) {
 };
 
 /**
- * Sets the display name for the thumb.
+ * Updates the display name component for the given video span id.
  */
-SharedVideoThumb.prototype.setDisplayName = function(displayName) {
+SharedVideoThumb.prototype.setDisplayName = function() {
```
Comment (saghul) [inline modules/UI/shared_video/SharedVideoThumb.js]: maybe rename it to `updateDisplayName` then?

File: modules/UI/shared_video/SharedVideoThumb.js [hunk 2]
```diff
@@ -65,9 +65,9 @@ SharedVideoThumb.prototype.createContainer = function(spanId) {
 };
 
 /**
- * Sets the display name for the thumb.
+ * Updates the display name component for the given video span id.
```
Comment (saghul) [inline modules/UI/shared_video/SharedVideoThumb.js]: There is no given span id. When I write "for the given" it usually refers to parameters, I find this comment confusing.

File: modules/UI/videolayout/LocalVideo.js [hunk 3]
```diff
@@ -87,9 +87,9 @@ LocalVideo.prototype.createContainer = function() {
 };
 
 /**
- * Sets the display name for the given video span id.
+ * Updates the display name component for the given video span id.
  */
-LocalVideo.prototype.setDisplayName = function(displayName) {
+LocalVideo.prototype.setDisplayName = function() {
```
Comment (saghul) [inline modules/UI/videolayout/LocalVideo.js]: Ditto

File: modules/UI/videolayout/RemoteVideo.js [hunk 4]
```diff
@@ -519,11 +519,9 @@ RemoteVideo.prototype.addRemoteStreamElement = function(stream) {
 };
 
 /**
- * Sets the display name for the given video span id.
- *
- * @param displayName the display name to set
+ * Updates the display name component for the given video span id.
  */
-RemoteVideo.prototype.setDisplayName = function(displayName) {
+RemoteVideo.prototype.setDisplayName = function() {
```
Comment (saghul) [inline modules/UI/videolayout/RemoteVideo.js]: Ditto

File: react/features/display-name/components/DisplayName.web.js [hunk 5]
```diff
@@ -144,6 +155,7 @@ class DisplayName extends Component<Props, State> {
         if (allowEditing && this.state.isEditing) {
             return (
                 <input
+                    autoComplete = 'off'
```
Comment (saghul) [inline react/features/display-name/components/DisplayName.web.js]: why? browsers usually remember your name and surname so it can be handy, can't it?

**Comment by zbettenbuk** [comment]
@virtuacoplenny not defaulting to some value caused some issues on mobile in the past. Can you please test mobile to see if that is not the case any more?

**Comment by virtuacoplenny** [comment]
> @virtuacoplenny not defaulting to some value caused some issues on mobile in the past. Can you please test mobile to see if that is not the case any more?

Based on the files changed, most of which are in the legacy jquery code, I don't think mobile has to be worried. I also default to an empty string, if that makes a difference.

**Pull request merged**